modules/android/vsandroid_vcxproj.lua

```diff
@@ -16,6 +16,25 @@
 	local config = p.config
 
 
+--
+-- Utility functions
+--
+
+	function table.findAndRemove(t, value)
+		for i, v in ipairs(t) do
+    		if v == value then
+      			table.remove(t, i)
+				break
+    		end
+  		end
+	end
```

Comment:
This should go in table.lua
There needs to be regression tests for its functionality
The indentation is inconsistent

Reply:
Shouldn't I do that in another PR?

Reply:
@samsinsane would have the final word on that one.
My feelings is that since this is a new utility function written for a specific use case, it would make sense to be added in this PR to give it context for anyone who later asks 'when/why was this put in'.

But I dont have the final word on that, so I'll defer to sam =)

Reply:
Right, thanks for the review either way.

Reply:
i looked into `table.lua` now and saw you can use `table.difference` instead.

```lua
-- instead of
-- table.findAndRemove(elements, vc2010.characterSet)
-- table.findAndRemove(elements, vc2010.wholeProgramOptimization)
-- table.findAndRemove(elements, vc2010.windowsSDKDesktopARMSupport)
-- do
elements = table.difference(elements, {
  vc2010.characterSet,
  vc2010.wholeProgramOptimization,
  vc2010.windowsSDKDesktopARMSupport,
})
-- or
elements = table.filter(elements, function(entry) {
  local tbl = {
    [vc2010.characterSet] = true,
    [vc2010.wholeProgramOptimization] = true,
    [vc2010.windowsSDKDesktopARMSupport] = true,
  }
  return not tbl[entry]
})
```

both of them do not change in-place but copy the table.

i suggest you use one of those for now and in a separate standalone pull request you (or me or anyone else) adds the in-place removal to `table.lua` and changes those calls to your call - but you should really think about allowing a table and maybe also a function (beware of clashes, as you can't remove a single function without wrapping it in a table then) as input instead of just a single element.

Reply:
@WorldofBay I guess `table.difference` will do it for now - and feel free to do it yourself :)